src/components/notification/channel/lark.ts

```diff
@@ -0,0 +1,159 @@
+                  {
+                    tag: 'text',
+                    text: `New ${notifType} from Monika\n`,
+                  },
+                  {
+                    tag: 'text',
+                    text: `${message.summary}\n`,
+                  },
+                ],
+              ],
+            },
+          },
+        },
+      }
+      break
+    case 'incident':
+    case 'recovery':
+      larkMessage = {
+        msg_type: 'post',
+        content: {
+          post: {
+            en_us: {
+              title: 'Monika Notification',
+              content: [
+                [
+                  {
+                    tag: 'text',
+                    text: `New ${notifType} from Monika\n`,
+                  },
+                  {
+                    tag: 'text',
+                    text: `Message: ${message.summary}\n`,
+                  },
+                  {
+                    tag: 'a',
+                    text: `URL: ${message.meta.url}\n`,
+                    href: `${message.meta.url}`,
+                  },
+                  {
+                    tag: 'text',
+                    text: `Time: ${message.meta.time}\n`,
+                  },
+                  {
+                    tag: 'text',
+                    text: `From: ${message.meta.monikaInstance}`,
+                  },
+                ],
+              ],
+            },
+          },
+        },
+      }
+      break
+    case 'status-update':
+      larkMessage = {
+        msg_type: 'post',
+        content: {
+          post: {
+            en_us: {
+              title: 'Monika Status Update',
+              content: [
+                {
+                  tag: 'a',
+                  text: `${message.meta.monikaInstance}`,
+                  href: `${message.meta.monikaInstance}`,
+                },
+                {
+                  tag: 'text',
+                  value: `${message.meta.numberOfProbes}`,
+                },
+                {
+                  tag: 'text',
+                  value: `${message.meta.averageResponseTime} ms in the last 24 hours`,
+                },
+                {
+                  tag: 'text',
+                  value: `${message.meta.numberOfIncidents} in the last 24 hours`,
+                },
+                {
+                  tag: 'text',
+                  value: `${message.meta.numberOfRecoveries} in the last 24 hours`,
+                },
+                {
+                  tag: 'text',
+                  value: message.meta.numberOfSentNotifications,
+                },
+              ],
+            },
+          },
+        },
+      }
+      break
+    default:
+      break
+  }
+
+  try {
+    const res = await axios({
+      method: 'POST',
+      url: data.url,
+      headers: {
+        Accept: 'application/json',
+        'Content-type': 'application/json',
+      },
+      data: larkMessage,
+    })
+
+    return res
+  } catch (error) {
+    throw error
```

Comment:
If in the end we will throw the same error, why do we need to have try/catch in here?
It would be simpler to not use try/catch instead.

Reply:
thanks, great feedback. updated.